aea/cli_gui/__init__.py

```diff
@@ -664,14 +655,25 @@ def favicon():
     return app
 
 
+def _terminate_process(process: subprocess.Popen):
+    """Try to process gracefully."""
+    poll = process.poll()
+    if poll is None:
+        # send SIGTERM
+        process.terminate()
+        try:
+            # wait for termination
+            process.wait(3)
```

Comment:
this `wait` without the `try` caused some tests to fail. Now we force the termination with SIGKILL if the `SIGTERM` didn't work.